### ISSUE TITLE
bibtex inproceedings & "series" from event/acronym, at least in ACM pubs

### ISSUE THREAD

**Issue by jowens** [front-matter/commonmeta-py#46]
This is a matter of taste, but when I typeset my own @inproceedings, I follow many bib providers and encode the abbreviation for the conference in the BibTeX `series` field. This is the abbreviated name of the conference and is much handier to web-search than searching on an entire conference title.

e.g.: ACM: my paper doi:10.1145/3225058.3225122 was published at "ICPP 2018". In the input JSON this is available as `event/acronym`. I would personally add the field `series = ICPP 2018`.

I don't see this as a particularly consistent thing across publishers but I would like to see it in the generated BibTeX if possible.

**Comment by mfenner** [comment]
I agree and this is how I have implemented (in CrossRef XML, will implement in CrossRef JSON as well).
